specs/system_info.table

```diff
@@ -0,0 +1,8 @@
+table_name("system_info")
+description("System information for identification.")
+schema([
+    Column("hostname", TEXT, "Hostname"),
```

Comment:
It might be worth mentioning in the description that this is the network hostname, which includes the host's name and its default search domain.